closure/bin/build/jscompiler.py

```diff
@@ -97,12 +97,16 @@ def _GetJsCompilerArgs(compiler_jar_path, java_version, source_paths,
   # Add the application JAR.
   args += ['-jar', compiler_jar_path]
 
-  for path in source_paths:
-    args += ['--js', path]
+  #collect list of compiler flags
+  compiler_flags.extend(['--js ' + path for path in source_paths])
+  if jvm_flags:
+    compiler_flags.extend(jvm_flags)
```

Comment:
I'm confused. Why are you re-adding `jvm_flags` to the arg list? This is redundant as they're added above.